cloud/resource_pulsar_cluster.go

```diff
@@ -84,6 +84,13 @@ func resourcePulsarCluster() *schema.Resource {
 				Description:  descriptions["location"],
 				ValidateFunc: validateNotBlank,
 			},
+			"release_channel": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				Default:      "rapid",
```

Comment:
Is the rapid channel used by default?

Reply:
yes, on UI also uses the rapid as default

Reply:
yes, keep consistent with console UI